README.md

```diff
@@ -154,42 +154,38 @@ type MultiVersionInfo struct {
 
 ## Known issues
 
-To run container-diff on image IDs or URLs, docker must be installed.
+To run container-diff on image IDs, docker must be installed.
 
 ## Example Run
 
+// TODO FIX EXAMPLE RUN
```

Comment:
Oops, that was a note to self that I addressed but forgot to take out.  I did verify by rerunning the example and the output currently there is the most recent.